packages/app/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "start": "backstage-cli app:serve",
-    "build": "backstage-cli app:build",
+    "bundle": "backstage-cli app:build",
```

Comment:
still a bit unsure why this one isn't built like the rest

Reply:
Because I don't want yarn build in the root to unconditionally trigger the build in the app. It's because the build-cache command isn't being used for the app, as it doesn't make sense there.

If this wasn't there the app would be rebuilt every time, which is a bit of a waste. We would actually want to run tsc checks though, but might also wait and see if stuff breaks first xP